src/Resources/contao/classes/Backend.php

```diff
@@ -302,11 +302,11 @@ protected function getBackendModule($module, PickerInterface $picker = null)
 		// Dynamically add the "personal data" module (see #4193)
 		if (\Input::get('do') == 'login')
 		{
-			$arrModule = array('tables'=>array('tl_user'), 'callback'=>'ModuleUser');
+			$arrModule = array('tables'=>array('tl_user'));
 		}
 
 		// Check whether the current user has access to the current module
-		elseif ($module != 'undo' && !$this->User->hasAccess($module, 'modules'))
+		if ($module != 'undo' && !$this->User->hasAccess($module, 'modules'))
```

Comment:
Strange that `undo` is here but `login` is now in `hasAccess`. But I guess that's up to another refactor.

Reply:
I also find this strange and would prefer to handle the login module check here instead of hardcoded in the `BackendUser::hasAccess()` method. @Toflar Do you think it is possible?

Reply:
Like so? fd47c7224868466fcc1db0f09cbe5834062ab752

Reply:
Hmm let me check if we can get that even nicer

Reply:
Well, it's not really possible without adding more keys to the `$GLOBALS['BE_MOD']`. In fact we could extend that array with an `doNotDisplay` and `disablePermissionCheck` key. That way everything would be supported, also for third party modules.

Reply:
Here we go, that way it's also supported for any third party module which is even better I find :) https://github.com/contao/core-bundle/pull/1559/commits/4531f7eaaf99d0e6e49bc83860c4b20ffa137dbf

Reply:
I agree. :)